__tests__/Pipeline.integration.js

```diff
@@ -96,6 +96,33 @@ describe('integration tests', () => {
       const firstHit = response.hits.hits[0]
       expect(firstHit._source.title[0]).toEqual(resourceTitle)
     })
+
+    const searchExpectations = [
+      { phrase: resourceTitle, totalHits: 1},
+      { phrase: 'cøol tītlé', totalHits: 1},
+      { phrase: 'cool title', totalHits: 1},
+      { phrase: 'cöôl title', totalHits: 1},
+      { phrase: 'COOL title', totalHits: 1},
+      { phrase: 'cool', totalHits: 1},
+      { phrase: 'title', totalHits: 1},
+      { phrase: 'coooool tiiiitle', totalHits: 0},
+    ]
```

Comment:
initially my `searchExpectations` list contained:

```javascript
{ phrase: 'COOL TITLE', totalHits: 1}
```

but that failed.  as you can see from line 105, having one of the words in all caps works.  i'm still to elasticsearch naïve to have a guess as to why things behaved this way.  happy to try and figure out that behavior if it seems wrong and the cause isn't obvious to others.

Reply:
fixed in #78, i think